deploy/olm-catalog/akka-cluster-operator/0.2.3/akka-cluster-operator-certified.v0.2.3.clusterserviceversion.yaml

```diff
@@ -0,0 +1,332 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: akka-cluster-operator-certified.v0.2.3
+  namespace: placeholder
+  annotations:
+    capabilities: Seamless Upgrades
+    categories: Application Runtime
+    certified: 'true'
+    containerImage: 'registry.connect.redhat.com/lightbend/akka-cluster-operator-certified:0.2.2'
```

Comment:
Does this need to move to v0.2.3?

Reply:
No, there are no changes to the operator itself. This is all only because of the demo app change to a   certified image.